config/application.yml.example

```diff
@@ -18,3 +18,6 @@ EVSS_CLAIMS_BASE_URL: 'http://csraciapp6.evss.srarad.com:7003/wss-claims-service
 EVSS_DOCUMENTS_BASE_URL: 'http://csraciapp6.evss.srarad.com:7003/wss-document-services-web-3.1/rest/'
 EVSS_SAMPLE_CLAIMANT_USER: '{"first_name": "Jane","last_name": "Doe","edipi": "1105051936","participant_id": "123456789"}'
 MVI_URL: 'http://ps-dev.commserv.healthevet.va.gov:8110/psim_webservice/IdMWebService'
+
+# For CORS requests; separate multiple origins with a comma
+WEB_ORIGIN: 'http://localhost:3000,http://localhost:3001'
```

Comment:
no - on dev, staging and prod, since the AJAX requests will be coming from a different origin from the API (www.vets.gov vs api.vets.gov) we'll need to specify these per environment.